deisctl/units/deis-logspout.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=deis-logspout
-Requires=docker.socket etcd.service
-After=docker.socket etcd.service
+Wants=docker.socket etcd2.service
```

Comment:
Do we really want to change this to `Wants`? The logspout component won't function without `docker.socket`, so I think it'll need to remain `Requires`, no?